app/screens/settings/notification_settings_email/notification_settings_email_base.js

```diff
@@ -84,25 +84,25 @@ export default class NotificationSettingsEmailBase extends PureComponent {
         }
     };
 
-    setEmailNotifications = (value) => {
-        const {sendEmailNotifications} = this.props;
-
-        let email = 'false';
-        if (sendEmailNotifications && value !== Preferences.INTERVAL_NEVER.toString()) {
-            email = 'true';
-        }
-
-        this.setState({
-            email,
-            newInterval: value,
-        });
+    setEmailInterval = (value) => {
```

Comment:
Renamed the function `setEmailNotifications` to `setEmailInterval` to make it sound like what it's actually doing.